packages/perennial-oracle/contracts/ChainlinkFeedOracle.sol

```diff
@@ -155,7 +173,7 @@ contract ChainlinkFeedOracle is IOracleProvider {
     function _versionToPhase(uint256 version) private view returns (Phase memory phase) {
         uint256 phaseId = _latestPhaseId();
         phase = _phases[phaseId];
-        while (uint256(phase.startingVersion) > version) {
+        while (phase.startingRoundId == 0 || uint256(phase.startingVersion) > version) {
```

Comment:
is there not a phase 0?

Reply:
Both phases and rounds are 1-indexed in Chainlink, so checking that the startingRoundId is basically checking if the phase is empty (and therefore invalid)